evaluation/benchmarks/needle_in_haystack/__init__.py

```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: Copyright (c) 1993-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Could you also add inits for infinite and longbenchv1/v2? They are currently missing (not really important, but makes it uniform).